modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectLayoutLocalServiceTest.java

```diff
@@ -467,7 +467,7 @@ public void testUpdateObjectLayout() throws Exception {
 		ObjectLayoutTab objectLayoutTab1 = _addObjectLayoutTab();
 
 		ObjectLayout objectLayout = _objectLayoutLocalService.addObjectLayout(
-			TestPropsValues.getUserId(),
+			RandomTestUtil.randomString(), TestPropsValues.getUserId(),
```

Comment:
It would be nice to assert the externalReferenceCode used to update the ObjectLayout.